modules/features/player/src/main/java/au/com/shiftyjelly/pocketcasts/player/view/playercontrols/PlayerControls.kt

```diff
@@ -0,0 +1,234 @@
+import com.airbnb.lottie.compose.LottieAnimation
+import com.airbnb.lottie.compose.LottieCompositionSpec
+import com.airbnb.lottie.compose.rememberLottieAnimatable
+import com.airbnb.lottie.compose.rememberLottieComposition
+import com.airbnb.lottie.compose.rememberLottieDynamicProperties
+import com.airbnb.lottie.compose.rememberLottieDynamicProperty
+import kotlin.time.Duration
+import kotlin.time.DurationUnit
+import kotlin.time.toDuration
+import kotlinx.coroutines.launch
+import au.com.shiftyjelly.pocketcasts.localization.R as LR
+
+@Composable
+fun PlayerControls(
+    playerViewModel: PlayerViewModel,
+) {
+    val playerControlsData by playerViewModel.listDataLive
+        .map {
+            PlayerControlsData(
+                it.podcastHeader.isPlaying,
+                it.podcastHeader.theme,
+                it.podcastHeader.skipBackwardInSecs.toDuration(DurationUnit.SECONDS),
+                it.podcastHeader.skipForwardInSecs.toDuration(DurationUnit.SECONDS),
+            )
+        }
+        .observeAsState(PlayerControlsData())
+
+    Content(
+        playerControlsData = playerControlsData,
+        onPlayPauseClick = { playerViewModel.onPlayPauseClicked() },
+        onSkipForwardClick = { playerViewModel.onSkipForwardClick() },
+        onSkipBackClick = { playerViewModel.onSkipBackwardClick() },
+        onSkipForwardLongPress = { playerViewModel.onSkipForwardLongClick() },
+    )
+}
+
+@OptIn(ExperimentalMaterialApi::class)
+@Composable
+private fun Content(
+    playerControlsData: PlayerControlsData,
+    onPlayPauseClick: () -> Unit,
+    onSkipForwardClick: () -> Unit,
+    onSkipBackClick: () -> Unit,
+    onSkipForwardLongPress: () -> Unit,
+) {
+    val tintColor = Color(ThemeColor.playerContrast01(playerControlsData.theme))
+    Row(
+        modifier = Modifier
+            .fillMaxWidth()
+            .wrapContentHeight()
+            .padding(bottom = dimensionResource(R.dimen.large_play_button_margin_bottom)),
+        horizontalArrangement = Arrangement.SpaceBetween,
+        verticalAlignment = Alignment.CenterVertically,
+    ) {
+        Spacer(
+            modifier = Modifier.size(8.dp),
+        )
+
+        SkipButton(
+            skipDuration = playerControlsData.skipBackInSecs,
+            contentDescription = stringResource(LR.string.skip_back),
+            tintColor = tintColor,
+            onClick = onSkipBackClick,
+        )
+
+        AnimatedPlayPauseButton(
+            isPlaying = playerControlsData.playing,
+            onClick = onPlayPauseClick,
+            iconTint = MaterialTheme.theme.colors.playerContrast06,
+            circleColor = tintColor,
+        )
+
+        SkipButton(
+            skipDuration = playerControlsData.skipForwardInSecs,
+            scaleX = -1f,
+            contentDescription = stringResource(LR.string.skip_forward),
+            tintColor = tintColor,
+            onClick = onSkipForwardClick,
+            onLongClick = onSkipForwardLongPress,
```

Comment:
Long press on the skip forward button to "Mark played" or "Next episode" didn't seem to work for me.

Reply:
I can't reproduce it 🤔 

https://github.com/user-attachments/assets/14be352f-a8a5-48bb-a4a3-d95a1f16586b